src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -542,14 +538,14 @@ class NetworkOPsImp final
     clock_type& m_clock;
     beast::Journal m_journal;
 
-    std::unique_ptr <LocalTxs> m_localTX;
+    std::unique_ptr <LocalTxs> m_localTX {make_LocalTxs()};
 
     std::recursive_mutex mSubLock;
 
     std::atomic<OperatingMode> mMode;
 
-    std::atomic <bool> mNeedNetworkLedger;
-    bool m_amendmentBlocked;
+    std::atomic <bool> mNeedNetworkLedger {false};
+    std::atomic <bool> m_amendmentBlocked {false};
```

Comment:
Prefer `amendmentBlocked_` instead (it's the "new" style). Since you're here, maybe rename the `mNeedNetworkLedger` bit too: `needNetworkLedger_` or somesuch?